extensions/amp-image-lightbox/0.1/amp-image-lightbox.js

```diff
@@ -903,7 +903,7 @@ class AmpImageLightbox extends AMP.BaseElement {
     this.sourceElement_ = sourceElement;
 
     // Initialize the viewer.
-    this.sourceImage_ = dom.elementByTag(sourceElement, 'img');
+    this.sourceImage_ = sourceElement.querySelector('.i-amphtml-fill-content');
```

Comment:
safest approach in my opinion is `img:not(.i-amphtml-intrinsic-sizer)`
another acceptable approach would be querying only direct children for `img` since the sizer `img` is in a separate `div`

Reply:
+1 to direct children only (see src/dom.js:childElementByTag)

Reply:
Oh, It is also working well. I add more tests and find what is the best way :)